pkgs/apps/mrcc/default.nix

```
@@ -22,8 +22,8 @@ stdenv.mkDerivation rec {
     in
     requireFile rec {
       name = "mrcc.${dashVersion}.binary.tar.gz";
-      sha256 = "59570fff722b80e1ca8ef46a8052c0569a8990120ad87f9fe0a180200a9729cc";
-      url = "https://www.mrcc.hu/index.php/download-mrcc/mrcc-binary/send/4-mrcc-binary/31-mrcc-${dashVersion}-binary-tar";
+      hash = "sha256-FNfIGSA3AuC2X2iOl9HiIt7mV+oNDqCpP04z6eaD228=";
```

```suggestion
      sha256 = "sha256-FNfIGSA3AuC2X2iOl9HiIt7mV+oNDqCpP04z6eaD228=";
```
This is how the `requireFile` override is written, or the download from `srcurl` fails.